forms-flow-web/src/components/Dashboard/Dashboard.js

```diff
@@ -129,8 +126,12 @@ const Dashboard = React.memo(() => {
     dispatch(setMetricsSubmissionPageChange(pageNumber));
   };
   const getFormattedDate = (date) => {
-    return moment.utc(date).format("YYYY-MM-DDTHH:mm:ssZ").replace(/\+/g, "%2B");
+    return moment
+      .utc(date)
+      .format("YYYY-MM-DDTHH:mm:ssZ")
```

Comment:
Can you make this date format configurable across

Reply:
yes this date formats will be changed to services for all will doing it in another ticket